monitoring/README.md

```diff
@@ -3,44 +3,101 @@
 <img src="images/ks-monitoring-setup.png" width="80%" height="80%" title="kubestellar-monitoring-setup">
 
 
+### Requirement
+To follow the instructions here, you need to have release `v0.2.0-rc12` or higher for the status-addon controller and agent installed in your enviroment. If you're using an older release, please follow these instructions to update your environment: 
+
+1.  Remove the status-addon-controller and status-addon-agent from your current environment:
+
+```bash
+helm --kube-context its1 -n open-cluster-management delete status-addon
+k --context its1 -n cluster1 delete manifestwork addon-addon-status-deploy-0
+```
+
+2. Re-deploy the status-addon controller and agents: 
+
+```bash
+helm --kube-context its1 upgrade --install ocm-status-addon -n open-cluster-management oci://ghcr.io/kubestellar/ocm-status-addon-chart --version v0.2.0-rc12
+```
+
 ### Description
-This example shows how to deploy monitoring tools (Prometheus, Grafana and Pyroscope) for KubeStellar control plane components - see architecture image above. These instructions can be used to deploy monitoring tools to any KubeStellar system. If you already have the monitoring tools installed in your environment, you can skip to `step-2`.
+This example shows how to deploy monitoring tools (Prometheus/Thanos, Grafana and Pyroscope) for KubeStellar control plane components - see architecture image above. These instructions can be used to deploy monitoring tools to any KubeStellar system. If you already have the monitoring tools installed in your environment, you can skip to `step-2`.
+
 
 #### 1. Install the monitoring tools:
 
 Starting from a local directory containing the git repo, do the following:
 
+a) Deploy the monitoring tools in the hosting cluster (e.g., `kind-kubeflex`):
+
+```bash
+cd monitoring
+./install-ks-monitoring.sh --cluster-context kind-kubeflex --set core
+```
+
+Optionally, check the deployment of the monitoring tools:
+
 ```bash
-$ cd monitoring
-$ ./install-ks-monitoring.sh
+kubectl --context kind-kubeflex -n ks-monitoring get pods
+```
+
+Output:
+```bash
+NAME                                                   READY   STATUS    RESTARTS   AGE
+grafana-7bf7b4c965-cwv4p                               1/1     Running   0          58s
+minio-0                                                1/1     Running   0          3m58s
+minio-1                                                1/1     Running   0          3m58s
+prometheus-kube-prometheus-operator-79d5cf844f-nbzvb   1/1     Running   0          2m57s
+prometheus-prometheus-kube-prometheus-prometheus-0     3/3     Running   0          2m54s
```

Comment:
FYI, I see 2/2 instead of 3/3 here when I test this.

Reply:
This is correct! Probably information from an old configuration. Just updated it with the output of the latest configuration that matches your output.